modules/migration-state-migration-cli.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/about-mtc-3-4.adoc
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+[id="migration-state-migration-cli_{context}"]
+= Migrating an application's state
+
+You can use {mtc-full} ({mtc-short}) to migrate an application's state. {mtc-short} copies persistent volume claims (PVCs) and specific Kubernetes objects, for example, `Secret` custom resources (CRs) or config maps that store an application's state.
+
+If you have a CI/CD pipeline, you can migrate stateless components by deploying them on the target cluster. Then you can migrate the state of the application by selecting PVCs and Kubernetes objects.
+
+You can migrate PV data from the source cluster to PVCs that are already provisioned in the target cluster by mapping PVCs in the `MigPlan` CR. This ensures that the target PVCs of migrated applications are synchronized with the source PVCs.
+
+After you have migrated PV data, you can migrate the Kubernetes objects.
+
+[id="excluding-pvcs_{context}"]
+== Excluding persistent volume claims
+
+You can exclude persistent volume claims (PVCs) by adding the `spec.persistentVolumes.pvc.selection.action` parameter to the `MigPlan` custom resource (CR) after the persistent volumes (PVs) have been discovered.
+
+.Prerequisites
+
+* `MigPlan` CR with PVs
+
+.Procedure
+
+* Add `spec.persistentVolumes.pvc.selection.action` to the `MigPlan` CR:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: <migplan>
+  namespace: openshift-migration
+spec:
+...
```

Comment:
We have a proposed change in formatting out to modify this `...` usage: https://github.com/openshift/openshift-docs/pull/35814/files

Reply:
`#...` looks weird. I'll change it if that PR is ever merged.